inc/menu-functions.php

```diff
@@ -30,7 +30,11 @@ function twenty_twenty_one_add_sub_menu_toggle( $output, $item, $depth, $args )
 	if ( 0 === $depth && in_array( 'menu-item-has-children', $item->classes, true ) ) {
 
 		// Add toggle button.
-		$output .= '<button class="sub-menu-toggle" aria-expanded="false" onClick="twentytwentyoneExpandSubMenu(this)"><span class="screen-reader-text">' . esc_html__( 'Toggle child menu', 'twentytwentyone' ) . '</span></button>';
+		$output .= '<button class="sub-menu-toggle" aria-expanded="false" onClick="twentytwentyoneExpandSubMenu(this)">';
```

Comment:
Should we make these icons available in the icon class again?
https://github.com/WordPress/twentytwentyone/blob/trunk/classes/class-twenty-twenty-one-svg-icons.php#L33

Reply:
I don't see why not... I'll push a commit with that in a minute

Reply:
Done